integration/Substra/README.md

```diff
@@ -0,0 +1,41 @@
+# Substra
+
+[Substra](https://docs.substra.org/en/latest/) is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
+
+Substra main usage is a production one. It has already been deployed and used by hospitals and biotech companies (see the [MELLODDY](https://www.melloddy.eu/) project for instance). Yet Substra can also be used on a single machine on a virtually splitted dataset for two use cases:
+
+debugging code before launching experiments on a real network
+
+performing FL simulations
+
+Substra was created by [Owkin](https://owkin.com/) and is now hosted by the [Linux Foundation for AI and Data](https://lfaidata.foundation/).
+
+# SubstraFL example on FLamby TCGA BRCA dataset
+
+This example illustrate the basic usage of SubstraFL, and propose a model training by Federated Learning
+using de Federated Average strategy on the TCGA BRCA dataset.
+
+The objective of this example is to launch a *federated learning* experiment on the six centers provided on Flamby
+(called organization on Substra), using the **FedAvg strategy** on the **baseline model**.
+
+This example does not use the deployed platform of Substra and will run in local mode.
+
+## Requirements
+
+- To run this example locally, please make sure to download and unzip in the same directory as this example the
+    assets needed to run it:
+
+    Please ensure to have all the libraries installed, a *requirements.txt* file is included in the zip file, where
+    you can run the command: `pip install -r requirements.txt` to install them.
```

Comment:
no zip folder here, you can directly say to do `pip install -r requirements.txt`

Reply:
Copy past too fast, thanks for the catch !